.gitignore

```diff
@@ -31,6 +31,9 @@ env/
 # Translations
 *.mo
 
+# IDE configs
+/.idea
```

Comment:
This path specification is weird, why do you need it? Wouldn't it be enough to add `*.idea`?